generate-schema.js

```diff
@@ -0,0 +1,22 @@
+// joi to json schema currently does not support v16 of joi
+const convert = require('joi-to-json-schema')
+const fs = require('fs')
+const { schema } = require('./lib/schema')
+const args = process.argv.slice(2) || []
+
+const theSchema = {
+  title: 'JSON schema for Release Drafter yaml files',
+  $id:
+    'https://github.com/release-drafter/release-drafter/blob/master/schema.json',
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  ...convert(schema())
+}
+
+// template is only required after deep merged, should not be required in the JSON schema
```

Comment:
I don't completely understand what this means. Could you elaborate?

Reply:
If this was required in the JSON schema it would show in your IDE/editor with a warning that `template` is missing.
Which it is not since it could be located in another repo via `_extends`

See: https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/.github/release-drafter.yml